profiles/BUILD

```diff
@@ -0,0 +1,16 @@
+package(
```

Comment:
Any reason not to put these rules in the root BUILD file? We're not going to distribute the profiles so this will be the only file ever in this dir.

Reply:
No real reason,  top level seemed a little cluttered and the package made the rules look clear e.g. `//profiles:santa_dev`.  

The other thought I had  was that we could just write `copy your provisioningprofiles in the profiles directory as Santa_dev and Santa_Daemon_Dev`. in future docs.

Reply:
Hmm, that latter reason is good enough for me